code/public/index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap"
```

Comment:
I think that you could have a bigger font weight, because the release date is barely readable on the overlay.